packages/calling-component-bindings/src/handlers/createCommonHandlers.ts

```diff
@@ -19,7 +19,7 @@ import { RemoteParticipant } from '@azure/communication-calling';
 import { CallSurvey, CallSurveyResponse } from '@azure/communication-calling';
 import { DtmfTone } from '@azure/communication-calling';
 import { AddPhoneNumberOptions } from '@azure/communication-calling';
-/* @conditional-compile-remove(teams-identity-support) */
+
```

Comment:
should we remove the empty lines or are we going to have code littered with random empty lines after we removed all of our conditional compilation flags?